### DISCUSSION TITLE
这个经验怎么提升

### DISCUSSION THREAD

**Discussion by JiuXiat** [share121/inter-knot#5056] [委托]
![Uploading Screenshot_2024-09-17-14-14-16-24_219376f1dbde3085d7f592166ad0f554.jpg…]()
我想把它的等级全部升满

**Comment by xiaqizhengzhao** [comment]
GitHub的项目，不过我建议你不要随便写一些没营养的东西。

**Comment by xiaqizhengzhao** [comment]
和你的项目数量有关
